packages/application-shell/src/components/user-settings-menu/user-settings-menu.js

```diff
@@ -40,28 +40,40 @@ UserAvatar.propTypes = {
   gravatarHash: PropTypes.string.isRequired,
   firstName: PropTypes.string,
   lastName: PropTypes.string,
+  email: PropTypes.string,
   // Injected
   handleMouseOver: PropTypes.func.isRequired,
   handleMouseOut: PropTypes.func.isRequired,
   isMouseOver: PropTypes.bool.isRequired,
 };
 const UserAvatarWithHoverState = withMouseOverState(UserAvatar);
 
-export default class UserSettingsMenu extends React.PureComponent {
+export class UserSettingsMenu extends React.PureComponent {
   static displayName = 'UserSettingsMenu';
 
   static propTypes = {
     firstName: PropTypes.string,
     lastName: PropTypes.string,
+    email: PropTypes.string,
     gravatarHash: PropTypes.string.isRequired,
+
+    // withRouter
+    location: PropTypes.shape({
+      pathname: PropTypes.string.isRequired,
+    }).isRequired,
   };
 
   render() {
+    const isAccountPath = Boolean(
+      matchPath(this.props.location.pathname, {
+        path: '/account',
+      })
+    );
     return (
       <div className={styles.container} data-test="user-settings-menu">
-        <MenuStateContainer>
+        <Downshift>
           {({ isOpen, toggleMenu }) => (
-            <div>
+            <div onBlur={toggleMenu}>
```

Comment:
Why are we not spreading the downshift props? That's what I meant a day ago to have better accessibility (e.g. click outside, keyboard navigation):

```js
<Downshift
    onChange={selection => alert(`You selected ${selection.value}`)}
    itemToString={item => (item ? item.value : '')}
  >
    {({
      getInputProps,
      getItemProps,
      getLabelProps,
      getMenuProps,
      isOpen,
      inputValue,
      highlightedIndex,
      selectedItem,
    }) => (
      <div>
        <label {...getLabelProps()}>Enter a fruit</label>
        <input {...getInputProps()} />
        <ul {...getMenuProps()}>
          {isOpen
            ? items
                .filter(item => !inputValue || item.value.includes(inputValue))
                .map((item, index) => (
                  <li
                    {...getItemProps({
                      key: item.value,
                      index,
                      item,
                      style: {
                        backgroundColor:
                          highlightedIndex === index ? 'lightgray' : 'white',
                        fontWeight: selectedItem === item ? 'bold' : 'normal',
                      },
                    })}
                  >
                    {item.value}
                  </li>
                ))
            : null}
        </ul>
      </div>
    )}
  </Downshift>,
```